lib/src/serialize.dart

```diff
@@ -598,9 +598,9 @@ void serializeStruct(Type self, SerialBuffer buffer, Object data,
     self.base.serialize(self.base, buffer, data,
         state: state, allowExtensions: allowExtensions);
   }
-  var dy = data as dynamic;
+  Map dy = data as dynamic;
   for (var field in self.fields) {
-    if (dy[field.name] != null) {
+    if (dy.containsKey(field.name)) {
```

Comment:
In eosjs serialize `if (field.name in data)` check if the map contain the field not if the field is not null. Deserializing a null value work fine but this was causing an error when serializing null value.